src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -333,6 +339,13 @@ ov::pass::StateManagementPattern::StateManagementPattern(ParameterVector& kv_par
                                                                           max_context_len.get_node_shared_ptr()};
         params.insert(params.end(), additional_params.begin(), additional_params.end());
 
+        if (use_block_indices_inputs) {
+            auto block_indices =
```

Comment:
I believe we need to set unique tensor name for each parameter
"block_indices" + id?

Reply:
Ok, I'll add it